### PR TITLE
vmss fix

### PR DESCRIPTION
**Pull request by hestolz**
Need to parse out instance number to match the schema where {instanceId} is just the VM's instance ID. 
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}/virtualmachines/{instanceId}

Below you can see the instanceId being properly parsed out
image

### CODE REVIEW

File: source/code/plugins/oms_configuration.rb [hunk 1]
```diff
@@ -147,7 +147,7 @@ def get_azure_resid_from_imds()
             if (imds_instance_json_compute['vmScaleSetName'].empty?)
               azure_resource_id = azure_resource_id + 'virtualMachines/' + imds_instance_json_compute['name']
             else
-              azure_resource_id = azure_resource_id + 'virtualMachineScaleSets/' + imds_instance_json_compute['vmScaleSetName'] + '/virtualMachines/' + imds_instance_json_compute['name']
+              azure_resource_id = azure_resource_id + 'virtualMachineScaleSets/' + imds_instance_json_compute['vmScaleSetName'] + '/virtualMachines/' + imds_instance_json_compute['name'].split('_')[-1]
```
Comment (simathih) [inline source/code/plugins/oms_configuration.rb]: is there an example of how the instance name with "_" looks like in VMSS?

Comment (hestolz) [reply source/code/plugins/oms_configuration.rb]: image

Comment (simathih) [reply source/code/plugins/oms_configuration.rb]: shouldn't the array index by 1 instead of -1 then?

Comment (hestolz) [reply source/code/plugins/oms_configuration.rb]: -1 gives the last element, 'name' is always the vmss name + '_' + instanceid, so -1 will always give us instanceId

Comment (simathih) [reply source/code/plugins/oms_configuration.rb]: looks good

**Pull request merged**